dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/ValidationUtils.java

```diff
@@ -97,7 +97,7 @@ public static List<Note> validateNotes(
       {
         // If a note having the same UID already exist in the db, raise
         // warning, ignore the note and continue
-        if (isNotEmpty(note.getNote()) && preheat.getNote(note.getNote()).isPresent()) {
+        if (isNotEmpty(note.getNote()) && preheat.noteExists(note.getNote())) {
```

Comment:
not sure if we stick to it but usually we use `is/has` prefixes for such methods, maybe `preheat.hasNote`?